<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubMenuExpenses</class>
 <widget class="QMainWindow" name="SubMenuExpenses">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(221, 234, 240);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinAndMaxSize</enum>
    </property>
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,1">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutMenuShopping" stretch="0,1">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="menuButtonShopping">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 0px;
text-align: left;
font: 700 16pt &quot;Verdana&quot;;

</string>
          </property>
          <property name="text">
           <string>Vásárlás</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutSubMenuShopping">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>15</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButtonShoppingReceipt">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>10000</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 0px;
font: 12pt &quot;Algerian&quot;;
text-align: left;</string>
            </property>
            <property name="text">
             <string>Blokk hozzáadása</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonShoppingItem">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>100000</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 0px;
font: 12pt &quot;Algerian&quot;;
text-align: left;</string>
            </property>
            <property name="text">
             <string>Tétel hozzáadása</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutMenuHouses" stretch="0,1">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="menuButtonHouses">
          <property name="styleSheet">
           <string notr="true">border: 0px;
text-align: left;
font: 700 16pt &quot;Verdana&quot;;</string>
          </property>
          <property name="text">
           <string>Ingatlanok</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutSubMenuHouses">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>15</number>
          </property>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutMenuChildren" stretch="0,1">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="menuButtonChildren">
          <property name="styleSheet">
           <string notr="true">border: 0px;
text-align: left;
font: 700 16pt &quot;Verdana&quot;;</string>
          </property>
          <property name="text">
           <string>Gyerekek</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutSubMenuChildren">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>15</number>
          </property>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutMenuCars" stretch="0,1">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="menuButtonCars">
          <property name="styleSheet">
           <string notr="true">border: 0px;
text-align: left;
font: 700 16pt &quot;Verdana&quot;;</string>
          </property>
          <property name="text">
           <string>Autók</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutSubMenuCars">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>15</number>
          </property>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="menuButtonCredits">
        <property name="styleSheet">
         <string notr="true">border: 0px;
text-align: left;
font: 700 16pt &quot;Verdana&quot;;</string>
        </property>
        <property name="text">
         <string>Hitelek</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="menuButtonHealthCare">
        <property name="styleSheet">
         <string notr="true">border: 0px;
text-align: left;
font: 700 16pt &quot;Verdana&quot;;</string>
        </property>
        <property name="text">
         <string>Orvos</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="menuButtonOther">
        <property name="styleSheet">
         <string notr="true">border: 0px;
text-align: left;
font: 700 16pt &quot;Verdana&quot;;</string>
        </property>
        <property name="text">
         <string>Egyéb</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
